dbms/src/Interpreters/Context.cpp

```diff
@@ -1650,8 +1650,10 @@ bool Context::initializeGlobalStoragePoolIfNeed(const PathPool & path_pool)
     auto lock = getLock();
     if (shared->global_storage_pool)
     {
-        // GlobalStoragePool may be initialized many times in some test cases for restore.
-        LOG_WARNING(shared->log, "GlobalStoragePool has already been initialized.");
+        // Can't init GlobalStoragePool twice.
+        // Because we won't remove the gc task in BackGroundPool
```

Comment:
Change back this comment pls. Also need change back to warning.

Reply:
We do NOT re-init GlobalStoragePool even in test cases now.

Reply:
okay, but the Notes, it's not right. need updated